stablehlo/reference/Element.cpp

```diff
@@ -81,6 +81,41 @@ std::complex<APFloat> getComplexValue(Element element) {
                                arryOfAttr[1].cast<FloatAttr>().getValue());
 }
 
+Element calcWithUpcastToDouble(
```

Comment:
Another thing that is interesting about related work is that function parameters are not limited to just functions but are instead callables. I think this extra bit of generality would be useful here as well.

Reply:
Thanks! I've updated PR in response to #190